### DISCUSSION TITLE
Vercel says Invalid Configuration for my domain and asks to add a type A, but this type A is already present.

### DISCUSSION THREAD

**Discussion by guillaume-g** [vercel/community#6174] [Help]
### Summary

Hi,

Sorry to bother you folks, but my domain started malfunctioning out of nowhere.
I checked my account and found out that it is domain-related. 

On my domain setting /settings/domains in my personal account I have a "Invalid Configuration". It says I need to add a type A. But : 
- It was already working since January
- I didn't change anything
- The type A and the address it asks to add is already present.

Did anyone have already deal with an issue like this ? I don't even know where to start.

Please ask more questions, I am quick to reply !

Thanks in advance,

### Example

yaudev.ch

### Steps to Reproduce

No step to reproduce as it is domain related with Vercel interface.

**Comment by amyegan** [comment]
Hey @guillaume-g. What is the domain?

**Reply by guillaume-g** [reply]
The domain is yaudev.ch and I have login in Vercel with this GitHub account. Thanks in advance :)

**Comment by sami5001** [comment]
@amyegan I am also having the exact same issue. My domain is `hcaibootcamp.site`. The domain is from Namecheap. Thanks a lot!

**Comment by amyegan** [comment]
Hi there! I recommend using our [domain troubleshooting guide](https://vercel.com/docs/projects/domains/troubleshooting#troubleshooting-domains) for this. The [Common DNS issues](https://vercel.com/docs/projects/domains/troubleshooting#common-dns-issues) section will be of use for checking which records are present. 

Sometimes regional DNS server differences can affect domains. You can check the global propagation status of the records with a tool like [whatsmydns.net](https://www.whatsmydns.net).

https://vercel.com/docs/projects/domains/troubleshooting#troubleshooting-domains

**Reply by sami5001** [reply]
Thanks a lot @amyegan! Following the steps, I could not find any issues, so contacted the domain registration company. Now found out the there is a serverhold on this. So resolving that now. I'm sure this will then work. Thanks for your help that led to narrowing down the issue.
